src/app/WriteClient.h

```diff
@@ -136,15 +136,15 @@ class WriteClient : public Messaging::ExchangeDelegate
     CHIP_ERROR FinalizeMessage(System::PacketBufferHandle & aPacket);
 
     /**
-     *  Once SendWriteRequest returns successfully, the WriteClient will
+     *  Once SendWriteRequestMessage returns successfully, the WriteClient will
      *  handle calling Shutdown on itself once it decides it's done with waiting
      *  for a response (i.e. times out or gets a response). Client can specify
      *  the maximum time to wait for response (in milliseconds) via timeout parameter.
      *  Default timeout value will be used otherwise.
-     *  If SendWriteRequest is never called, or the call fails, the API
+     *  If SendWriteRequestMessage is never called, or the call fails, the API
      *  consumer is responsible for calling Shutdown on the WriteClient.
      */
-    CHIP_ERROR SendWriteRequest(SessionHandle session, System::Clock::Timeout timeout);
+    CHIP_ERROR SendWriteRequestMessage(SessionHandle session, System::Clock::Timeout timeout);
```

Comment:
No, this might be sending multiple messages.  This should stay `SendWriteRequest`.

Reply:
sure, rename it back.